### ISSUE TITLE
Staging Review finding: Unclear to screen reader users when search is complete

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99001]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Representation Management
**Product:** Appoint a Representative MVP
**Feature:** Appoint a Representative + Representative Status Widget

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/appoint-rep/representative-select](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/representative-select), when you enter a search term and press Search, it takes a moment for results to load. When the search is complete, the results are visible beneath the search tool and you can proceed with selecting your rep.

But for screen reader users, the experience breaks down. While waiting for results, the loading period is announced as "Finding accredited representatives, loading progress bar" --- so far so good. When the results do load, however, there's no announcement made to screen reader users. Without the visual cues that something on the page has changed, screen reader users have to guess. Has enough time passed? Is it still searching? Will it ever be done? etc.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:
1. Navigate to /appoint-rep/representative-select.
2. Activate a screen reader (recommend NVDA on Windows or VoiceOver on MacOS for quick testing)
3. Enter a search term.
4. Press tab to move to the search button, then press enter.
5. Note that the loading message is announced, but once results are displayed there's no message to screen reader users.

</details>

## Recommended action

Two options here:
1. Move focus to some kind of summary message (Showing 5 results for "[term]") above the results, similar to what's done on the Find a Rep tool. Moving focus to that message will ensure that it's announced to screen reader users immediately.
2. Add an `aria-live` region to announce to screen reader users when the results are loaded.

Between those options I prefer (1), but either is fine.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by oddball-lindsay** [comment]
Fixed! More context in the ticket linked above.
